### DISCUSSION TITLE
How can we create a bigger community for Plugins

### DISCUSSION THREAD

**Discussion by matthijs166** [payloadcms/payload#865] [Feature Requests & Ideas]
I love Payload and think we can build a bigger community around plugins for payload.
There are not a lot of plugins for payload but it would be nice to have an overview of them.

Plugins I'm currently using:
- nestedDocs
- webp
- s3-upload
- SEO
- payload-hash-upload
- payload-blurhash-plugin

**Comment by denolfe** [comment]
Hey @matthijs166 , this is a great idea - one that we've been thinking about quite a bit. We're playing around with a few ideas, but I think we're leaning toward making a dedicated page on the site for now - then something more robust in the future.

**Reply by denolfe** [reply]
We're also looking at ensuring plugins use the `payload-plugin` topic on their repo, so they will show on this page: https://github.com/topics/payload-plugin
